test/testdata/exhaustive/exhaustive.go

```diff
@@ -0,0 +1,10 @@
+package exhaustive
```

Comment:
I used a separate package here to check that the linter is integrates correctly for cross-package checks (ie., switch statement and enum definitions in different packages).

Reply:
This is no longer true. I've reverted to using the same file because importing the testdata package did not seem to work.